lib/puppet/provider/cisco_ace/nxapi.rb

```diff
@@ -1,9 +1,9 @@
 #
 # The NXAPI provider for cisco_ace.
 #
-# July, 2015
+# January 2016
```

Comment:
We'll need the noise cleaned up before we can merge this PR.  Please remove the following commits from your PR:

* bdf0e0e3e20e12195663edfcadd29b8c9dc5d027
* 89621e2e3e97bc673105fd6e2a447d5cdfc59272

I recommend you checkout `release_1.2.0` branch on your fork, pull in only your doc change commits, then submit a new PR.